src/elexmodel/handlers/data/Estimandizer.py

```diff
@@ -0,0 +1,153 @@
+import numpy as np
+
+
+class Estimandizer:
+    """
+    Estimandizer. Generate estimands explicitly.
+    """
+
+    def __init__(self, data_handler, estimands):
+        self.data_handler = data_handler
+        self.estimands = estimands
+        self.transformations = []
+        self.transformation_map = {
+            "margin": [self.calculate_margin],
```

Comment:
Are these functions Lenny gave you to include?

Reply:
Lenny mentioned margin, percent of votes, and candidate name vote / two party vote. I added a few more that I could come up with